File: core/exercises/01_env_vars/solution/cpp/main.cpp
#include <cstdlib>
#include <iostream>
#include <string>

int main() {
    const char* value = std::getenv("APP_NAME");
    if (value == nullptr) {
        std::cout << "APP_NAME=MISSING\n";
    } else {
        std::cout << "APP_NAME=" << value << "\n";
    }
    return 0;
}

File: core/exercises/01_env_vars/solution/python/main.py
"""Exercise 01 — Environment variables."""

import os


def main() -> None:
    value = os.environ.get("APP_NAME")
    if value is None:
        print("APP_NAME=MISSING")
    else:
        print(f"APP_NAME={value}")


if __name__ == "__main__":
    main()
